README.md

```diff
@@ -7,23 +7,26 @@
 
 **Table of Contents**
 
+- [What is CockroachDB](#what-is-cockroachdb)
 - [Status](#status)
-- [Running Cockroach Locally](#running-cockroach-locally)
-- [Deploying Cockroach in production](#deploying-cockroach-in-production)
+- [Running CockroachDB Locally](#running-cockroach-locally)
+- [Deploying CockroachDB in production](#deploying-cockroach-in-production)
 - [Getting in touch and contributing](#get-in-touch)
 - [Talks](#talks)
 - [Design](#design) and [Datastore Goal Articulation](#datastore-goal-articulation)
 - [Architecture](#architecture) and [Client Architecture](#client-architecture)
 
-[![WIRED on CockroachDB](/resource/doc/wired-preview.png?raw=true)](http://www.wired.com/2014/07/cockroachdb/)
+## What is CockroachDB
+
+CockroachDB is a distributed SQL database built on top of a key:value datastore. It was designed to support ACID transactional semantics and versioned values as first-class features. The primary design goal is global consistency and survivability, hence the name. CockroachDB aims to tolerate disk, machine, rack, and even datacenter failures with minimal latency disruption and no manual intervention. CockroachDB nodes (RoachNodes) are symmetric; a design goal is homogenous deployment (one binary) with minimal configuration.
```

Comment:
@vivekmenezes fixed. 

@xiang90 Don't feel this it the right place for a link to the blog post, but will look for a better place to link to it.